src/app/login/login.component.scss

```diff
@@ -0,0 +1,63 @@
+@import '../../mixins.scss';
+@import '../../variables.scss';
+
+:host {
```

Comment:
If you use :host angular will add an additional selector to css so the bundled css will be larger.